.pre-commit-config.yaml

```
@@ -84,6 +86,20 @@ repos:
     hooks:
       - id: shellcheck
         args: ["--severity=warning"]
+  - repo: https://github.com/rapidsai/pre-commit-hooks
+    rev: v0.6.0
+    hooks:
+      - id: verify-copyright
+        files: |
+          (?x)
+              [.](cmake|cpp|cu|cuh|h|hpp|sh|pxd|py|pyx)$|
+              CMakeLists[.]txt$|
+              meta[.]yaml$
```

```suggestion
              recipe[.]yaml$
```

I know this is only showing up in the diff because you're moving it but now that you are... this should be changed to `recipe.yaml`.

`meta.yaml` files were only present here when this project's conda packages were build with `conda build`. Now that they're built with `rattler-build`, the recipes are in files name `recipe.yaml`: https://github.com/NVIDIA/cuopt/blob/branch-25.08/conda/recipes/cuopt/recipe.yaml